coalib/results/result_actions/IgnoreResultAction.py

```diff
@@ -65,3 +67,49 @@ def apply(self, result, original_file_dict, file_diff_dict, language: str,
             file.writelines(ignore_diff.modified)
 
         return file_diff_dict
+
+    def get_ignore_comment(self, origin, language):
+        r"""
+        Returns a string of Ignore Comment, depending on the language
+        Supports Single Line Comments
+
+        >>> IgnoreResultAction().get_ignore_comment("Bear", "css")
+        '/* Ignore Bear */\n'
+
+        And Multiline Comments
+
+        >>> IgnoreResultAction().get_ignore_comment("Bear", "c")
+        '// Ignore Bear\n'
+
+        >>> import sys
+        >>> import logging
+        >>> logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)
+        >>> # doctest: +ELLIPSIS
+        ... IgnoreResultAction().get_ignore_comment("Bear", "Dothraki")
+        WARNING:root:coala does not support Ignore in "Dothraki"...
+
```

Comment:
I don't understand... `pytest -k "Ignore"` runs the test correctly. ( I added a deliberate error and it noticed it.) 
But this error is only occuring in `pytest` not in `pytest -k "Ignore"`